src/__tests__/__snapshots__/renderer.test.tsx.snap

```diff
@@ -472,7 +472,7 @@ exports[`slack jsx renders inline text elements with promises returning text ele
 Object {
   "elements": Array [
     Object {
-      "text": "Hello, world! <@foo>,<@bar>",
+      "text": "Hello, world! <@foo><@bar>",
```

Comment:
this was slightly wrong before anyway, as i never explicitly said i wanted a comma in the list.